src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -12,12 +12,16 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import org.springframework.test.annotation.DirtiesContext;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@DirtiesContext
 @DisplayName("지하철역 관련 기능")
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
 public class StationAcceptanceTest {
+    String stationName1 = "강남역";
```

Comment:
```suggestion
    String 강남역 = "강남역";
```

이런식으로 직관적으로 해도 좋을것 같아요 😄